ui.frontend/src/view/FormContainer.js

```diff
@@ -116,6 +116,14 @@ export default class FormContainer {
         }
     }
 
+    getFormId() {
+        return this._model._jsonModel.id;
+    }
+
+    getFormTitle() {
+        return this._model._jsonModel.title;
```

Comment:
this._model.title
_jsonModel is an internal object. we should avoid using that.

Reply:
@barshat7 Please change this iteratively

Reply:
yes FORMS-7159 is raised for the same